src/shell/mod.rs

```diff
@@ -408,13 +408,23 @@ impl WorkspaceSet {
             return Err(InvalidWorkspaceIndex);
         }
 
+        // Animate if workspaces overview isn't open
+        let layer_map = layer_map_for_output(&self.output);
+        let animate = !layer_map
+            .layers()
+            .any(|l| l.namespace() == WORKSPACE_OVERVIEW_NAMESPACE);
```

Comment:
How weird is this interaction with the touchpad gestures? I imagine it feels quite bad when the workspace abruptly changes. We probably just want to disable the gestures for now, when the overview is active?